src/tasks/minions/monsterActivity.ts

```diff
@@ -199,30 +200,50 @@ export const monsterTask: MinionTask = {
 		const isOnTaskResult = await isOnSlayerTask({ user, monsterID, quantityKilled: quantity });
 
 		const superiorTable = isOnTaskResult.hasSuperiorsUnlocked && monster.superior ? monster.superior : undefined;
-		const isInCatacombs = !usingCannon ? monster.existsInCatacombs ?? undefined : undefined;
+		const isInCatacombs = (!usingCannon ? monster.existsInCatacombs ?? undefined : undefined) && !isInWilderness;
+
+		let hasRingOfWealthI = user.gear.wildy.hasEquipped('Ring of wealth (i)') && isInWilderness;
+		if (hasRingOfWealthI) {
+			messages.push('\nYour clue scroll chance is doubled due to wearing a Ring of Wealth (i).');
```

Comment:
Just a comment (im not sure), if the newlines are really needed in the messages added in this pr.